pkg/splunk/enterprise/util.go

```diff
@@ -1928,3 +1928,88 @@ func migrateAfwFromPhase2ToPhase3(ctx context.Context, client splcommon.Controll
 func isAppFrameworkMigrationNeeded(afwStatusContext *enterpriseApi.AppDeploymentContext) bool {
 	return afwStatusContext != nil && afwStatusContext.Version < currentAfwVersion && len(afwStatusContext.AppsSrcDeployStatus) > 0
 }
+
+// updateCRStatus fetches the latest CR, and on top of that, updates latest status
+func updateCRStatus(ctx context.Context, client splcommon.ControllerClient, dirtyCR splcommon.MetaObject) {
+	reqLogger := log.FromContext(ctx)
+	scopedLog := reqLogger.WithName("updateCRStatus").WithValues("name", dirtyCR.GetName(), "namespace", dirtyCR.GetNamespace())
+
+	//sgontla: Fix the UT to accommodate multiple retries
+	for i := 0; i < 1; i++ {
```

Comment:
can we fix retryCount at least to be 10 times , also make it const

Reply:
Yes Vivek. Some of the UTs GET calls are expecting the max. number of retries. I will be pushing one more change.